### ISSUE TITLE
Call to a member function getBody() on null GuzzleHttpAdapter.php:132 from Laravel job.

### ISSUE THREAD

**Issue by alexsnkr** [DigitalOceanPHP/DigitalOceanV2#229]
I'm using https://github.com/GrahamCampbell/Laravel-DigitalOcean.

When I use it in a Laravel job I get this error:
`Call to a member function getBody() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getBody() on null at xxx\\vendor\\toin0u\\digitalocean-v2\\src\\Adapter\\GuzzleHttpAdapter.php:132)`

This is the job handle() method:
```
public function handle()
{
    Log::info(DigitalOcean::connection('main')->region()->getAll());
}
```
Running this from the controller works fine.

**Comment by AndreFabris** [comment]
Same issue here. Upgraded to Laravel 5.7, guzzlehttp/guzzle 6.3, graham-campbell/digitalocean 5.1 and it has stopped working

**Comment by elegasoft** [comment]
I was running into the same issue as well while running jobs in my testing environment. I've found that the root error in my case is actually a `cURL error 60: SSL certificate problem: unable to get local issuer certificate (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)` which causes the `$response` to be `null` in `function handleError()`. 

Having followed the accepted answer [StackOverflow -- curl-error-60-ssl-certificate-in-laravel-5-4](https://stackoverflow.com/questions/42094842/curl-error-60-ssl-certificate-in-laravel-5-4) the error stopped occurring for me.

**Comment by DavidAnderson684** [comment]
See this error all the time (not using Laravel). Happens whenever there is an HTTP timeout. https://github.com/toin0u/DigitalOceanV2/blob/aa3fbe5efba26632665230829aef5ada4fa415cb/src/Adapter/GuzzleHttpAdapter.php#L132 assumes that `$this->response` is a response object, but sometimes it's null.

**Comment by DavidAnderson684** [comment]
The easiest way to reproduce is to set an absurdly low timeout, e.g. `$client = new Client(['timeout' => 0.1, 'headers' => ['Authorization' => sprintf('Bearer %s', $api_key)]]);`

**Comment by GrahamCampbell** [comment]
~Moving to part of the v4 design issue https://github.com/DigitalOceanPHP/DigitalOceanV2/issues/249. I am planning to smash out a fresh package for v4 which will feel like the old package, but have none of the issues and all of the compatibility. ;)~ EDIT: See comment below.

**Comment by GrahamCampbell** [comment]
Update: Actually, I think I have fixed this in the 3.0 branch now, so this will be included in the 3.0.0 release. I think this error was happening when there was some kind of transport error resulting in an exception from guzzle which did not contain a response.
